components/ProgressView/examples/ProgressViewExample.m

```diff
@@ -194,11 +210,11 @@ - (void)setupConstraints {
     @"H:|-(p)-[coloredView]-(p)-|",
     @"H:|-(p)-[backwardResetView]-(p)-|",
     @"H:|-(p)-[backwardAnimateView]-(p)-|",
-    @"H:|-(>=p)-[stockLabel]-(>=p)-|",
-    @"H:|-(>=p)-[tintedLabel]-(>=p)-|",
-    @"H:|-(>=p)-[coloredLabel]-(>=p)-|",
-    @"H:|-(>=p)-[backwardResetLabel]-(>=p)-|",
-    @"H:|-(>=p)-[backwardAnimateLabel]-(>=p)-|",
+    @"H:|-(p)-[stockLabel]-|",
```

Comment:
This change will stretch the width of the label, relying on the text alignment for positioning. Is this what you want, or do should we allow the UILabel to hug its content?

Reply:
I'm fine with it the way it is. Do you have a preference?